.github/CODEOWNERS

```diff
@@ -4,4 +4,10 @@
 
 # By default, unless overridden later,
 # Wolf should review every policy change:
-cloud-aws/src/main/resources/definitions/   @hortonworks/cloudbreak-wolf
\ No newline at end of file
+cloud-aws/src/main/resources/definitions/   @hortonworks/cloudbreak-wolf
+
+# Raven team should review authorization framework related changes
```

Comment:
Best thing ever. Not a serious comment but would b great to even check every controller change.